Frameworks/CoreData/NSAttributeDescription.mm

```diff
@@ -14,8 +14,111 @@
 //
 //******************************************************************************
 
+#import <CoreData/CoreData.h>
+#import <Foundation/Foundation.h>
 #import <StubReturn.h>
-#import <CoreData/NSAttributeDescription.h>
+
+#import <CoreData/NSManagedObjectModel-XMLParsing.h>
+
+@interface NSAttributeDescription () {
+    StrongId<id> _minValue;
+    StrongId<id> _maxValue;
+}
+@end
```

Comment:
I thought the hotness was to put this in the implementation? Also curious that _ things are publicly exposed.